sim/team-validator.js

```diff
@@ -901,7 +901,8 @@ class Validator {
 			alreadyChecked[template.speciesid] = true;
 			if (dex.gen === 2 && template.gen === 1) tradebackEligible = true;
 			// STABmons hack to avoid copying all of validateSet to formats
-			if (ruleTable.has('ignorestabmoves') && format.noLearn.indexOf(moveid) < 0 && !move.isZ) {
+			let noLearn = format.noLearn || dex.getFormat('gen7stabmons').noLearn;
+			if (ruleTable.has('ignorestabmoves') && noLearn.indexOf(move.name) < 0 && !move.isZ) {
```

Comment:
> `noLearn.indexOf(move.name) < 0`

...standard is to use `!noLearn.includes(move.name)`

Reply:
Ah my bad, this is just how Sketchmons did it so I copied, will make a PR to change it

Reply:
Change Sketchmons too, if it still exists.

Reply:
It doesn’t, stabmons replaced it